packages/client/mutations/EndRetrospectiveMutation.ts

```diff
@@ -47,10 +47,6 @@ graphql`
           }
         }
       }
-      settings {
-        recallBotId
-        videoMeetingURL
-      }
```

Comment:
@nickoferrall it's unclear why these are here. I think it should be a fragment if this is necessary.
You could replace it with
```
team {
  meetingSettings(type: retrospective) {
    ...
  }
}
if necessary
```